tools/dep_updaters/update-openssl.sh

```diff
@@ -10,24 +10,40 @@ cleanup() {
 }
 
 download() {
+
+  NEW_VERSION=$1
+
   if [ -z "$1" ]; then
-    echo "Error: please provide an OpenSSL version to update to"
-    echo "	e.g. ./$0 download 3.0.7+quic1"
-    exit 1
+    NEW_VERSION="$("$NODE" --input-type=module <<'EOF'
```

Comment:
How is it supposed to work on Node.js 16 which uses OpenSSL 1.x?

Reply:
Its not what this pr is doing